docs/scenarios/library-authoring/pinning-libraries-to-rc2.md

```diff
@@ -0,0 +1,70 @@
+# How to manage package dependency versions for RC2
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers what you need to know about package versions for authoring libraries.
+
+## Glossary
+
+**Metapackage** - A NuGet package that represents a set of NuGet packages.  They're organized in a way that makes sense for general library and app development.
+
+**Trimming** - The act of removing the packages you do not depend on from a metapackage.  The end result is that your library only depends on packages it uses.  See guidance in the libraries doc for more information.
```

Comment:
Currently that guidance doesn't exist.  I'll track updating this with a link when it's written.